src/common/models/Projection.ts

```diff
@@ -481,13 +481,16 @@ export class Projection {
     );
 
     return dailyPositives.map((dailyPositive, idx) => {
-      const positive = dailyPositive || 0;
-      const negative = dailyNegatives[idx] || 0;
-      const total = positive + negative;
+      const positive = dailyPositive;
+      const negative = dailyNegatives[idx];
```

Comment:
Note: Previously the values were never null, so we didn't actually hit the `|| 0` case.